crates/egui-wgpu/src/renderer.rs

```diff
@@ -345,6 +345,7 @@ impl Renderer {
                     compilation_options: wgpu::PipelineCompilationOptions::default()
                 }),
                 multiview: None,
+                cache:None,
```

Comment:
Huh, how come the format checker doesn't complain about the lack of space here...? 🤔

Reply:
strange indeed, I poked it to format that file directly and it didn't care. hum.